README.md

```diff
@@ -0,0 +1,118 @@
+
+1. Basic knowledge of Python to get started. Be able to install jupyterlab on your machine and be up and running.
+2. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like `trimesh` or `meshio`.
+3. Advanced users will be familiar with GUI frameworks like Qt, VTK, and advanced data science.
+
+### Tutorial Outline.
+
+0. Getting Started - PyVista for 3D Visualization within Python. (10 min for talk, 10 min for exercise)
+
+1. Basic usage - Reading and plotting 3D data using built-in mesh data and external files.  (10 min for talk, 10 min for exercise)
+
+2. What is a Mesh? - Learn about the PyVista data types and how you can use it to visualize almost anything. (15 min for talk, 10 min for exercise)
+
+3. PyVista Data Model - Learn about the data model of the PyVista framework and how you can visualize your data in style using point, cell, and field data. (15 min for talk, 10 min for exercise)
+
+4. Break. Stretch fingers and grab some coffee. (15 minutes)
+
+5. Mesh animation - Show how to animate meshes using the PyVista API to make awesome online documentation. (10 min for talk, 10 min for exercise)
+
+6. PyVista, jupyterlab, and sphinx - Leverage PyVista to make some awesome interactive web documentation. (20 min for talk, 10 min for exercise)
+
+7. Quick break. Prepare for the final run. (10 minutes)
+
+8. PyVista & VTK - Show how PyVista uses VTK and how you can combine the best of both worlds! (10 minutes for talk, 10 minutes for exercise)
+
+9. PyVista and Qt - Demonstrate how to use PyVista to create standalone applications using pyinstaller and the Qt framework. (15 min for talk, 10 for exercise)
+
+10. PyVista in Action - Show how PyVista is already being used within several projects and can be used for all things visualization. (15 min for talk)
+
+11. Open up to Questions. (15 minutes)
+
+### Additional Tutorial Information.
+
+We will base the material for the tutorial on [the examples in PyVista's online documentation](https://docs.pyvista.org/examples/index.html).
+[Transform 2021: Guide to PyVista Tutorial](https://github.com/banesullivan/transform-2021) will also be used as material.
+The tutorial itself will be in the [pyvista-tutorial](https://github.com/pyvista/pyvista-tutorial) repository.
+
+### Tutorial Prerequisites.
+
+We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+1. Basic knowledge of Python to get started. Be able to install jupyterlab on your machine and be up and running.
+2. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like `trimesh` or `meshio`.
+3. Advanced users will be familiar with GUI frameworks like Qt, VTK, and advanced data science.
```

Comment:
See suggested change for this above.

Reply:
See https://github.com/pyvista/pyvista-tutorial/pull/1/files#r806605033